public/error/scripts/main.js

```diff
@@ -0,0 +1,7 @@
+var tryAgainButton = document.querySelector(".try-again") || null;
```

Comment:
You don't need `|| null;` since it will return `null` if it doesn't find the element.